pkg/cmd/template/regular_input.go

```diff
@@ -101,5 +108,10 @@ func (s *RegularFilesSource) Output(out TemplateOutput) error {
 	s.ui.Debugf("### result\n")
 	s.ui.Printf("%s", combinedDocBytes) // no newline
 
+	if len(nonYamlFileNames) > 0 {
+		s.ui.Warnf("\n"+`Warning: Non-YAML templates are not rendered to standard output.
+If you want to include those results, use the --output-files or --dangerous-emptied-output-directory flag.
+Non-YAML files are: [%s]`+"\n", strings.Join(nonYamlFileNames, ", "))
```

Comment:
whats should we do when there is a lot of files? imagine one yaml template that uses data.read of 100 csv files. we would end up with a enormous warning and in this case use knows that it's perfectly fine.

Reply:
We have decided to remove the files as part of the warning message. Perhaps this can be revisited in the future with some kind of verbose option, the message by itself seems okay for now.